docs/source/install_rocm.rst

```diff
@@ -17,20 +17,52 @@ The following ROCm libraries are required:
 
   $ sudo apt install hipblas hipsparse rocsparse rocrand rocthrust rocsolver rocfft hipcub rocprim rccl
 
+Environment Variables
+---------------------
+
+When running CuPy for ROCm, the following environment variables need to be set.
+
+* ``HCC_AMDGPU_TARGET``: ISA name supported by your GPU.
```

Comment:
According to the ongoing discussion https://github.com/cupy/cupy/issues/4719#issuecomment-785981214 it seems we don't have to set `HCC_AMDGPU_TARGET` at runtime.